stages/07.extra-scripts/examples/extra-script-example.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# kuiper2.0 - Embedded Linux for Analog Devices Products
+#
+# Copyright (c) 2024 Analog Devices, Inc.
+# Author: Larisa Radu <larisa.radu@analog.com>
+
+
+# This script is run inside 'chroot'. 
+# - It works as if Kuiper is running on a system with a few limitations. 
+# - Check 'chroot' documentation for more informations.
+# This script is run as root, there is no need to use 'sudo' command.
+# Current directory is '/' (root) of the Kuiper rootfs.
+# If a file needs to be copied, it should be placed inside 'adi-kuiper-gen'.
+# If a variable from the configuration file is needed, it should be passed as a parameter at script calling.
+# - modify stages/07.extra-scripts/run.sh to pass parameters (see example)
+# - assign parameter value to a variable name
+# At this stage the Kuiper image is not yet partitioned. In order to modify what will be in the boot partition access /boot folder.
+# This script will not be in the resulted image. If this is necessary it should be copied manually.
+
```

Comment:
Just add here a kind of title that next lines are examples for different use cases.